lib/result-view.js

```diff
@@ -1,238 +1,273 @@
+
+        this.resultContainer = document.createElement('div');
+        this.resultContainer.classList.add('bubble-result-container');
+        outputContainer.appendChild(this.resultContainer);
+
+        this.errorContainer = document.createElement('div');
+        this.errorContainer.classList.add('bubble-error-container');
+        outputContainer.appendChild(this.errorContainer);
+
+        this.statusContainer = document.createElement('div');
+        this.statusContainer.classList.add('bubble-status-container');
+        this.spinner = this.buildSpinner();
+        this.statusContainer.appendChild(this.spinner);
+        outputContainer.appendChild(this.statusContainer);
+
+        let richCloseButton = document.createElement('div');
+        richCloseButton.classList.add('rich-close-button', 'icon', 'icon-x');
+        richCloseButton.onclick = () => this.destroy();
+        this.element.appendChild(richCloseButton);
+
+        let actionPanel = document.createElement('div');
+        actionPanel.classList.add('bubble-action-panel');
+        this.element.appendChild(actionPanel);
+
+        let closeButton = document.createElement('div');
+        closeButton.classList.add('action-button',
+            'close-button', 'icon', 'icon-x');
+        closeButton.onclick = () => this.destroy();
+        actionPanel.appendChild(closeButton);
+
+
+        let padding = document.createElement('div');
+        padding.classList.add('padding');
+        actionPanel.appendChild(padding);
+
+        let copyButton = document.createElement('div');
+        copyButton.classList.add('action-button',
+            'copy-button', 'icon', 'icon-clippy');
+        copyButton.onclick = () => {
+            atom.clipboard.write(this.getAllText());
+            return atom.notifications.addSuccess('Copied to clipboard');
+        };
+        actionPanel.appendChild(copyButton);
+
+        let openButton = document.createElement('div');
+        openButton.classList.add('action-button',
+            'open-button', 'icon', 'icon-file-symlink-file');
+        openButton.onclick = () => {
+            let bubbleText = this.getAllText();
+            return atom.workspace.open().then(editor => editor.insertText(bubbleText));
+        };
+        actionPanel.appendChild(openButton);
+
+        this.setMultiline(false);
+
+        this.tooltips = new CompositeDisposable();
+        this.addCopyTooltip(copyButton);
+        this.tooltips.add(atom.tooltips.add(openButton,
+            {title: 'Open in new editor'})
+        );
+
+        this._hasResult = false;
+        this._executionCount = null;
+
+        return this;
+    }
+
+    addCopyTooltip(element) {
+        return this.tooltips.add(atom.tooltips.add(element, {
+            title: () => {
+                if (this._executionCount === null) {
+                    return 'Copy to clipboard';
+                } else {
+                    return `Copy to clipboard (Out[${this._executionCount}])`;
```

Comment:
My misunderstanding! I missed that `addResult` is only invoked with results for a given request (and thus a unique execution count).